spec/error_mapper_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+RSpec.describe JsonapiErrorsHandler::ErrorMapper do
+  let(:subject) { described_class }
+
+  before(:each) {
+    subject.class_variable_set(:@@mapped_errors, {}) 
+  }
+
+  describe '.mapped_errors' do
+    it 'returns all the current mapped errors' do
+      expect(subject.mapped_errors).to eq({})
```

Comment:
You set class_variable few lines above, before every single test example, so you don't check how it'll behave without that `before` block.

Reply:
@swilgosz the `before` block will be redundant in case I already add the initialization for `@@mapped_errors` in `ErrorMapper` class. 
If there is no initialization for `@@mapped_errors`, and I try to check how it behaves without `before` block, it will raise `uninitialized class variable @@mapped_errors in JsonapiErrorsHandler::ErrorMapper` error.

To fix this, I will remove the `before` block and just keep the initialization for `@@mapped_errors`. How do you think?